testapi.pm

```diff
@@ -949,6 +952,38 @@ sub script_output($;$) {
     return $output;
 }
 
+
+=head2 share_tmp_file
+
+  share_tmp_file($relpath, $content)
+
+Saves content to the file in the worker pool directory using relative path,
+to be fetched via http later on using <autoinst_url>/files/#relpath/#file url.
+Can be used to modify files for specific test needs, e.g. autoinst profiles.
+Dies if cannot open file for writing.
+
+Example:
+  share_tmp_file(autoyast/autoinst.xml, "<profile>Test</profile>")
+
+
+=cut
+
+sub share_tmp_file($$) {
```

Comment:
Please just use `sub share_tmp_file` without the argument signature, we want to avoid it for … perl reasons :)

Reply:
We have it for other functions and we have 2 mandatory parameters. But fine with me